app/webpack/observations/show/components/activity_item.jsx

```diff
@@ -15,496 +16,618 @@ import UserImage from "../../../shared/components/user_image";
+        </Panel>
       </div>
     );
   }
-  const byClass = viewerIsActor ? "by-current-user" : "by-someone-else";
-  let footer;
-  if ( item.disagreement && !hideDisagreement ) {
-    const previousTaxonLink = (
-      <SplitTaxon
-        taxon={item.previous_observation_taxon}
-        url={urlForTaxon( item.previous_observation_taxon )}
-        target={linkTarget}
-        user={config.currentUser}
-      />
-    );
-    const footerText = I18n.t( "user_disagrees_this_is_taxon", {
-      user: ReactDOMServer.renderToString( userLink ),
-      taxon: ReactDOMServer.renderToString( previousTaxonLink )
-    } );
-    footer = (
-      <span
-        className="title_text"
-        dangerouslySetInnerHTML={{
-          __html: `* ${footerText}`
-        }}
-      />
-    );
-  }
-  if ( item.implicitDisagreement ) {
-    const footerText = I18n.t( "user_disagrees_with_previous_finer_identifications", {
-      user: ReactDOMServer.renderToString( userLink )
-    } );
-    footer = (
-      <span
-        className="title_text"
-        dangerouslySetInnerHTML={{
-          __html: `* ${footerText}`
-        }}
-      />
-    );
-  }
-  const elementID = isID ? `activity_identification_${item.id}` : `activity_comment_${item.id}`;
-  const itemURL = isID ? `/identifications/${item.id}` : `/comments/${item.id}`;
-  return (
-    <div id={elementID} className={`ActivityItem ${className} ${byClass}`}>
-      <div className="icon">
-        { ( !item.hidden || canSeeHidden || viewerIsActor ) && (
-          <UserImage user={item.user} linkTarget={linkTarget} />
-        ) }
-      </div>
-      <Panel className={panelClass}>
-        <Panel.Heading>
-          <Panel.Title>
-            <span className="title_text" dangerouslySetInnerHTML={{ __html: header }} />
-            { headerItems }
-            <time
-              className="time"
-              dateTime={item.created_at}
-              title={moment( item.created_at ).format( "LLL" )}
-            >
-              <a href={itemURL} target={linkTarget}>{ relativeTime }</a>
-            </time>
-            <ActivityItemMenu
-              item={item}
-              observation={observation}
-              config={config}
-              deleteComment={deleteComment}
-              deleteID={deleteID}
-              restoreID={restoreID}
-              setFlaggingModalState={setFlaggingModalState}
-              linkTarget={linkTarget}
-              trustUser={trustUser}
-              untrustUser={untrustUser}
-              hideContent={hideContent}
-              unhideContent={unhideContent}
-            />
-          </Panel.Title>
-        </Panel.Heading>
-        <Panel.Body>
-          { taxonChange }
-          <div className="contents">
-            { contents }
-          </div>
-        </Panel.Body>
-        { footer ? <Panel.Footer>{ footer }</Panel.Footer> : null }
-      </Panel>
-    </div>
-  );
-};
+}
 
 ActivityItem.propTypes = {
+  inlineEditing: PropTypes.bool,
```

Comment:
Ideally this is temporary. 

The way that the `ObservationModal` in /identify works vs the observation show page is significant enough that I think it should be treated separately.

Mainly it comes down to `Activity`  component vs `DiscussionList` component. I'm not sure if there are already plans to do so, but maybe the older `DiscussionList` can be replaced with the newer `Activity` feed? Even if not, it feels like completing it for the latter while leaving the former with existing functionality is a good point to separate the work.

Reply:
Yeah, this is kind of an ongoing issue. Historically, Identify was our first React page and I made it. Later, we started using React and Redux a bit differently, consolidating actions and reducers into ducks, and still later Patrick made the obs detail page with a mostly-novel set of components. Since then I've worked some of Patrick's obs detail work into Identify, like `ActivityItem`s, but not everything, like the `Activity` component or the `callAPI` action. I think Identify should move in that direction, but it would be kind of a big overhaul and out of scope for this issue, so I think your approach here is good.